elasticsearch/_async/client/__init__.py

```diff
@@ -305,7 +305,6 @@ async def info(self, params=None, headers=None):
         "version_type",
         "wait_for_active_shards",
         request_mimetypes=["application/json"],
-        response_mimetypes=["application/json"],
```

Comment:
Hmm, it's weird that `response_mimetypes` isn't being generated? Can we figure out why that's happening?

Reply:
Yes, I will check that out.